Resources/doc/index.rst

```diff
@@ -58,20 +58,20 @@ Step 2: Enable the bundle
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Enable the bundle in the kernel:
+ 
+.. code-block:: php
```

Comment:
the Symfony documentation conventions are to use the ``::`` shortcut at the end of the previous paragraph when we have a paragraph ending with a ``:`` followed by a code-block in the default language (i.e. PHP): http://sphinx-doc.org/rest.html#source-code 
Can you change it ?